choreolib/src/main/java/choreo/auto/AutoFactory.java

```diff
@@ -145,7 +144,7 @@ public <SampleType extends TrajectorySample<SampleType>> AutoFactory(
     this.resetOdometry = resetOdometry;
     this.controller = controller;
     this.driveSubsystem = driveSubsystem;
-    this.useAllianceFlipping = useAllianceFlipping;
+    this.useAllianceFlipping = () -> useAllianceFlipping;
```

Comment:
This should probably be changed to a boolean as well

Reply:
I don't want to have to change the underlying behavior yet cuz its a really pervasive change and im tired. I just need the public facing api to stabilize on what it will end up being.